<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDlgConfig</class>
 <widget class="QDialog" name="QDlgConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>279</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>482</width>
    <height>279</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>482</width>
    <height>279</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>端口设置</string>
  </property>
  <widget class="QGroupBox" name="gbPortOptions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>端口选项</string>
   </property>
   <widget class="QComboBox" name="cbStopBit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>140</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>停止位</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbFlowCtrl">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>无</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>硬件</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Xon / Xoff</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>170</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>流控制</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>端口</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbPort">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>21</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM16</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>110</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据位</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbDataBit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>80</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>奇偶校验</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbParity">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No Parity ( 无)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Odd (奇)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Even (偶)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mark (标记)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Space (空格)</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cbBaudRate">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>50</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>75</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>110</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>134</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>150</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>56000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>76800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>128000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>256000</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>50</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbModemOptions">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>调制解调器选项</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据流控</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbDataFlowCtrl">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>21</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None (无)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RTS/CTS (硬件)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Xon/Xoff (软件)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RTS/CTS &amp; Xon/Xoff</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>51</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>自动应答</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbAutoReply">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>51</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>否</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>是</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbBufferOptions">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>缓存容量</string>
   </property>
   <widget class="QSpinBox" name="sbInBuffer">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>21</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999999</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>8192</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>输入缓存</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>51</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>输出缓存</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbOutBuffer">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>51</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999999</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>8192</number>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnOK">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
